### PR TITLE
Update Erlang + Elixir version issue #22

### PR DESCRIPTION
**Pull request by nelsonic**
as part of updating the `:crypto` functions #22 , we need to update to the latest Erlang + Elixir 
Once this PR passes and is merged, we can work on updating all the `crypto` invocations #23

### CODE REVIEW

**Comment by nelsonic** [comment]
Apparently the Travis Build is failing because Postgres is unavailable ... 🙄 
https://travis-ci.org/dwyl/phoenix-ecto-encryption-example/builds/618302627
image

```
17:53:37.425 [error] GenServer #PID<0.4481.0> terminating
** (DBConnection.ConnectionError) tcp connect (localhost:5432): connection refused - :econnrefused
    (db_connection) lib/db_connection/connection.ex:84: DBConnection.Connection.connect/2
    (connection) lib/connection.ex:622: Connection.enter_connect/5
    (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
Last message: nil
** (Mix) The database for Encryption.Repo couldn't be created: killed
The command "mix cover" exited with 1.
```

**Comment by nelsonic** [comment]
Build passes but apparently we _have_ to specify the version of Erlang/OTP if we want to get the latest one:
https://travis-ci.org/dwyl/phoenix-ecto-encryption-example/builds/618304834#L297
image

So I'm going to add the OTP version to `.travis.yml` 📝

**Comment by nelsonic** [comment]
@SimonLab please review and merge if OK. thanks. 👍

File: lib/encryption/aes.ex [hunk 1]
```diff
@@ -63,6 +63,7 @@ defmodule Encryption.AES do
   # as above but *asumes* `default` (latest) encryption key is used.
   @spec decrypt(any) :: String.t
   def decrypt(ciphertext) do
+    IO.inspect ciphertext, label: "ciphertext"
```
Comment (SimonLab) [inline lib/encryption/aes.ex]: Do you want to keep this IO.inspect to see what is passed to the function?

Comment (nelsonic) [reply lib/encryption/aes.ex]: Ah, good catch. I added that in to debug your question ... https://github.com/dwyl/app/issues/250#issuecomment-559568401
removing it now. 👍

**Pull request merged**